target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -200,6 +201,9 @@ ramips_setup_macs()
 		wan_mac=$(mtd_get_mac_ascii factory wanmac)
 		label_mac=$(cat /sys/class/ieee80211/phy1/macaddress)
 		;;
+	dlink,dir-878-a1)
+		wan_mac=$(mtd_get_mac_binary factory 0xe006)
+		;;
```

Comment:
Please merge this with the case above.

Reply:
I'm confused one is in ascii & the other in binary ?

Reply:
There is more than one case above.